presto-main/src/main/java/com/facebook/presto/operator/aggregation/BindableAggregationFunction.java

```diff
@@ -116,15 +114,12 @@ public InternalAggregationFunction specialize(BoundVariables variables, int arit
         List<Type> inputTypes = boundSignature.getArgumentTypes().stream().map(x -> typeManager.getType(x)).collect(toImmutableList());
         Type outputType = typeManager.getType(boundSignature.getReturnType());
 
-        AggregationFunction aggregationAnnotation = definitionClass.getAnnotation(AggregationFunction.class);
-        requireNonNull(aggregationAnnotation, "aggregationAnnotation is null");
```

Comment:
In later code it is moved to `AggregationFromAnnotationsParser.parseFunctionDefinitions`